Assets/Data/Localization/German/MenuLocalization.json

```
@@ -126,19 +126,19 @@
         },
         {
             "id": "settingsMenu.controls.topView",
-            "text": "Flugzeug-Ansicht"
+            "text": "Draufsicht"
         },
         {
             "id": "settingsMenu.controls.extraControls",
-            "text": "Extra Steuerungen"
+            "text": "Weitere Steuerungen"
         },
         {
             "id": "settingsMenu.controls.makeDayTime",
-            "text": "Setze zu Tag"
+            "text": "Mach es Tag"
         },
         {
             "id": "settingsMenu.controls.makeNightTime",
-            "text": "Setze zu Nacht"
+            "text": "Mach es Nacht"
```

```suggestion
            "text": "Nachtzeit"
```
Sounds a bit nicer too

---

Same here, maybe not clear enough

---

In my first translation try I had
```suggestion
            "text": "Zu Nacht wechseln"
```
In my opinion it sounds bettern tahn "Mach es Nacht" but its still not perfect

---

Same her I don't really know which one sounds better.